### PR TITLE
Find the value from the input or select from form element

### PR DESCRIPTION
**Pull request by liamkeaton**
## Feature Description
We need to find the value of the form element

## Link to Ticket / Card:
https://trello.com/c/e4KJFoBU/1216-paypal-payments-from-usa-and-can-are-hitting-payment-issues

## Has the necessary documentation been created / updated?
- [ ] Yes
- [X] Not required for this ticket

## Has this been given a review by Design/UX?
- [ ] Yes
- [X] Not required for this ticket

### CODE REVIEW

File: utils/form-element.js [hunk 1]
```diff
@@ -56,6 +56,14 @@ class FormElement {
 		this.inputs.forEach(input => input.disabled = false);
 	}
 
+	/**
+	 * Get the value of the form element
+	 * Assumes the first input or select is what's required
+	 */
+	value () {
+		return this.$el.querySelector('input,select').value;
```
Comment (remybach) [inline utils/form-element.js]: `textarea`?

Comment (liamkeaton) [reply utils/form-element.js]: We currently don't have any textareas in our forms and I don't think we will

File: utils/form-element.js [hunk 2]
```diff
@@ -56,6 +56,14 @@ class FormElement {
...
+	}
+
```
Comment (umbobabo) [inline utils/form-element.js]: This space can be probably removed.

File: utils/form-element.js (hunk 1 above)
Comment (umbobabo) [inline utils/form-element.js]:
```suggestion
		return this.$el.querySelector('input, select').value;
```

**Pull request merged**